app/controllers/seances_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SeancesController < ApplicationController
+  # GET /seances or /seances.json
+  def index
+    @seances = Seance.includes(:movie, :hall).all
+  end
+  # GET /seances/new
+  def new
+    @seance = Seance.new
+    @halls = Hall.all
+    @movies = Movie.all
+  end
+  # POST /seances or /seances.json
+  def create
+    @seance = Seance.new(seance_params)
```

Comment:
You don't have a variable or method `seance_params`. Have you run this code? 😉

Reply:
I checked it after PR and it didn't work so I added `seance_params` so now it works